core/jazz_ui/src/app/pages/environment-overview/env-overview-section.component.html

```diff
@@ -287,10 +287,10 @@
 		<textarea class="text-area" [disabled]="textarea"  (ngModelChange)="textChanged  && lineNumbers()"
 		 [(ngModel)]="yaml" id="yaml" name="yaml" (blur)='checkYaml()'></textarea>
 	   </div>
-		<div *ngIf="isCancel">
-		<p class = "validation" *ngIf="isvalid">Valid Yaml file.</p>
-		<p class = "validation" *ngIf="invalid">Invalid Yaml file.</p>
-		</div>
+	</div>
+	<div class="valid" *ngIf="isCancel">
+		<p class="validation" *ngIf="isvalid">Valid Yaml file.</p>
+		<p class="Invalidation" *ngIf="invalid">Invalid Yaml file.</p>
```

Comment:
Can you change class name to 'invalidation'?

Reply:
done